packages/@okta/vuepress-theme-prose/assets/css/okta/pages/oie-embedded-sdk.scss

```diff
@@ -4,4 +4,9 @@
     text-align: center;
   }
 
+  .common-image-format-vertical-margin {
```

Comment:
You've added this, but it doesn't seem to be used anywhere.

Reply:
This css style is used. One example is located here.: ../docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/go/integrationsteps.md. I found that places where the image is used has little bottom padding.

Reply:
**Before the css common-image-format-vertical-margin style applied:**
![image](https://user-images.githubusercontent.com/78044107/128803849-fa7ccfd1-baed-4ae5-964c-a3a0c5994d29.png)
**After the style applied:**
![image](https://user-images.githubusercontent.com/78044107/128803908-8c3b702e-251b-451c-a234-0fb661faedb1.png)

Reply:
OK, thanks for the explanation. This looks like a style that should just be applied to all images included inside body copy (as does the centering rule), in which case it'd be better to apply it to global CSS than have special rules for the OIE docs. Can you open a ticket under Developer Platform to investigate this at some point? We can probably do this ourselves, when the OIE docs quietens down a bit.

Reply:
Jira ticket https://oktainc.atlassian.net/browse/OKTA-419329 was added to investigate and potentially fix the issue.

Reply:
Perfect, thanks! Resolving.